dev/reference/garbage-collection/configuration.md

```diff
@@ -87,3 +87,22 @@ update mysql.tidb set VARIABLE_VALUE="24h" where VARIABLE_NAME="tikv_gc_life_tim
 
 - 手动设置 GC concurrency。要使用该参数，必须将 [`tikv_gc_auto_concurrency`](#tikv-gc-auto-concurrency) 设为 `false` 。
 - 默认值：2
+
+## 关于 GC 流程的说明
+
+从 3.0 起，由于分布式 GC 模式和并行 Resolve Locks 的支持，部分配置选项的作用发生了变化。可根据下表理解不同版本中这些配置的区别：
+
+| 版本/配置          |  Resolve Locks          |  Do GC  |
```

Comment:
```suggestion
| 版本/配置          |  Resolve Locks          |  执行 GC  |
```

Reply:
This is consistent with the name of the third phase of GC, which is documented here: https://pingcap.com/docs-cn/v3.0/reference/garbage-collection/overview/

Reply:
Got it.